app/src/main/java/com/xinto/opencord/domain/mapper/ApiDomainMapper.kt

```diff
@@ -168,17 +168,17 @@ fun ApiMessage.toDomain(): DomainMessage {
 }
 
 fun ApiMessagePartial.toDomain(): DomainMessageRegularPartial {
-    val domainAuthor = author.mapToPartial { it.toDomain() }
-    val domainAttachments = attachments.mapToPartial { attachments ->
+    val domainAuthor = author.map { it.toDomain() }
```

Comment:
Why is the method called map?

Reply:
maps a partials value to another enclosed partial

Reply:
mapToPartial made no sense